betainc.go

```diff
@@ -0,0 +1,23 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "github.com/gonum/mathext/internal/cephes"
+
+// Incbeta computes the value of the incomplete beta function of x.
+// The incomplete beta function is
+//  int_0^x u^(a-1) * (1-u)^(b-1) du
+func Incbeta(a, b float64, x float64) float64 {
+	return cephes.Incbet(a, b, x)
+}
+
+// Invincbeta computes the inverse of the incomplete beta function.
```

Comment:
```
Invincbeta evaluates the inverse of the incomplete beta function B(x;a,b).
It returns the x for which
 y = B(x;a,b).
The parameters a and b must be positive, otherwise Invincbeta will panic.
```